2021/day08/io/example.output

```diff
@@ -1,3 +1,2 @@
 26
 61229
-
```

Comment:
White space being the biggest annoyance as of yet this AoC🙃

Reply:
Hmm :thinking:

Reply:
Sorry . It should not be a problem in most cases, but I have done some mistakes in some of the `example.input` and `.output` files.